planning/autoware_costmap_generator/include/autoware/costmap_generator/costmap_generator.hpp

```diff
@@ -107,7 +107,8 @@ class CostmapGenerator : public rclcpp::Node
     sub_points_{this, "~/input/points_no_ground", autoware::universe_utils::SingleDepthSensorQoS()};
   autoware::universe_utils::InterProcessPollingSubscriber<PredictedObjects> sub_objects_{
     this, "~/input/objects"};
-  autoware::universe_utils::InterProcessPollingSubscriber<tier4_planning_msgs::msg::Scenario>
+  autoware::universe_utils::InterProcessPollingSubscriber<
+    autoware_internal_planning_msgs::msg::Scenario>
```

Comment:
autoware_utils is better?

Reply:
This will be covered in the other PR. 
https://github.com/autowarefoundation/autoware.universe/pull/10191